### DISCUSSION TITLE
Glide duplicating image

### DISCUSSION THREAD

**Discussion by ChrisButterworth** [statamic/cms#5060] [Q&A]
Using the Glide tag in two places in a template with the same image at the same dimensions both of which are using different URLs instead of using the same one - is there a way to serve the same image in both tags and not generate a new image every time?

Oddest thing is that on another page it doesn't have the same behaviour - using the same image multiple times.

Any advice?

**Comment by riasvdv** [comment]
I think this PR aims to solve that: https://github.com/statamic/cms/pull/4927

**Comment by jasonvarga** [comment]
Are the glide tags **identical**? Not just width and height, but everything?
What are the different URLs they're outputting?

**Reply by ChrisButterworth** [reply]
Both tags are completely identical - only difference is placement.

**Reply by jasonvarga** [reply]
Can you provide the template?
